templates/carecoordination/cda/cda-validate-results.html.twig

```diff
@@ -0,0 +1,29 @@
+{# Note we use the cda-validate-results and data-error-count as a means for html consumers to grab information from the html #}
+<div class='control-group cda-validate-results' data-error-count="{{ validation.xsd|length + validation.errorCount }}">
+    <h5 class='text-info' style='padding: 0 0;'>{{ 'Schema Definition Errors'|xlt}} - ({{ validation.xsd|length }})</h5><hr style='margin: 0 0 10px;' />
+    {% if validation.xsd|length <= 1 %}
+        <p class='text-success'>{{"Passed XSD testing."|xlt}}</p><hr style='margin: 0 0 10px;' />
+    {% else %}
+        {% for error in validation.xsd %}
+            <blockquote style='margin: 0 0 2px;padding: 0px 5px 0 5px;'>
+                <p class='text-error' style='font-size:12px;'>{{ error|text }}</p>
+            </blockquote><hr style='margin: 0 0 10px;' />
+        {% endfor %}
+    {% endif %}
+
+    <h5 class='text-info'>{{ 'Schematron Errors'|xlt }} - ({{ validation.errorCount }})</h5><hr style='margin: 0 0 10px;' />
```

Comment:
would pipe validation.errorCount through attr

Reply:
fixed

Reply:
looks like did in line 2 but not in line 14

Reply:
fixed, looked at all the escaping in the file and hopefully got everything this time.